public/html/debug.html

```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+<!doctype html>
```

Comment:
No need to shout!

Reply:
Indeed, think prettier did this automatically and I thorough agree with the change